meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-common.cmd.in

```diff
@@ -46,6 +44,8 @@ if fiovb init ${devnum} && test -n "${board_is_closed}"; then
 	if test ! $? -eq 0; then run bootcmd_bootenv; fiovb write_pvalue bootfirmware_version "${bootfirmware_version}"; fi
 	fiovb read_pvalue debug 4
 	if test ! $? -eq 0; then fiovb write_pvalue debug 0; fi
+	fiovb read_pvalue is_secondary_boot 4
+	if test ! $? -eq 0; then fiovb write_pvalue is_secondary_boot 0; fi
```

Comment:
I don't think this write will work as we don't have is_secondary_boot at PERSIST_VALUE_LIST.

Reply:
Good catch, https://github.com/foundriesio/optee_os/pull/73